plugins/ruby-foreman-discovery/debian/rules

```diff
@@ -16,10 +16,14 @@ build:
 	cp $(PLUGIN).rb /usr/share/foreman/bundler.d/
 	cd /usr/share/foreman && ( \
 		/usr/bin/foreman-ruby /usr/bin/bundle install --local && \
+		/usr/bin/npm install --no-audit --no-optional --unsafe-perm && \
 		/usr/bin/foreman-ruby /usr/bin/bundle exec rake plugin:apipie:cache[$(PLUGIN)] cache_part=resources \
 			OUT=/var/lib/foreman/public/apipie-cache/plugin/$(PLUGIN) RAILS_ENV=development \
 			DATABASE_URL=nulldb://nohost FOREMAN_APIPIE_LANGS=en_US \
```

Comment:
@ezr-ondrej on line 22 we already have the `DATABASE_URL=nulldb://nohost`, is this what you were referring to?